presto-parser/src/main/java/io/prestosql/sql/SqlFormatter.java

```diff
@@ -1219,6 +1220,28 @@ protected Void visitInsert(Insert node, Integer indent)
             return null;
         }
 
+        @Override
+        protected Void visitUpdate(Update node, Integer indent)
+        {
+            builder.append("UPDATE ")
+                    .append(node.getTable().getName());
+
+            builder.append("\n")
+                    .append(indentString(indent + 1))
+                    .append("SET ");
+            builder.append(node.getAssignments().stream()
+                    .map(assignment -> format("%s = %s", assignment.getName().getValue(), formatExpression(assignment.getValue())))
+                    .collect(joining(", ")));
+            builder.append("\n")
```

Comment:
We should only output a blank line if there is a `WHERE`
```java
if (node.getWhere().isPresent()) {
    builder.append("\nWHERE ").append(...);
}

Reply:
Changed.